### ISSUE TITLE
Set 'en' as the admin language for demo projects

### ISSUE THREAD

**Issue by khchine5** [jrief/django-shop#68]
Hi @jrief ,
It would be more easy for a starter to get a demo project with 'en' as the admin language.
The part of fixtures file to fix in the file 'myshop.json'
```
{
  "model": "cms.usersettings",
  "pk": 1,
  "fields": {
    "user": [
      "admin"
    ],
    //"language": "de", 
      "language": "en", 
    "clipboard": 1
  }
},
```
Thanks.

**Comment by jrief** [comment]
Ups, oversaw this.
Is this still true?

**Comment by khchine5** [comment]
Hi @jrief ,
Yes , it is still there.
File : myshop.json ,line 99

**Comment by rfleschenberg** [comment]
@khchine5 can you send a PR against awesto/django-shop?

**Comment by khchine5** [comment]
Yes sure @rfleschenberg  , but the fixtures used by 'initialize_shop_demo' is available in awesto/django-shop. That is why I haven't create a PR.

**Comment by rfleschenberg** [comment]
Ah, I see! Where is that management command defined? I can't find it.

**Comment by rfleschenberg** [comment]
Never mind, I got it.

**Comment by jrief** [comment]
Stupid me! Didn't notice, when I created that fixture. Thanks for reporting.

@rfleschenberg could you please review PR #463. With that fix I'd like to release 0.9.4
